activerecord/test/cases/tasks/mysql_rake_test.rb

```diff
@@ -48,11 +56,17 @@ def test_establishes_connection_to_database
       ActiveRecord::Tasks::DatabaseTasks.create @configuration
     end
 
+    def test_when_database_created_successfully_outputs_info_to_stdout
+      $stdout.expects(:puts).with("Created database 'my-app-db'").once
+
+      ActiveRecord::Tasks::DatabaseTasks.create @configuration
+    end
+
     def test_create_when_database_exists_outputs_info_to_stderr
-      $stderr.expects(:puts).with("my-app-db already exists").once
+      $stderr.expects(:puts).with("Database 'my-app-db' already exists").once
```

Comment:
We should use StringIO for stderr also, same as stdout. Nice to change at the same time.